search/base.py

```diff
@@ -0,0 +1,18 @@
+"""
```

Comment:
i would not expect to find our base test case class for ElasticSearch-related functionality, or any test case class for that matter, in `search/base.py`. i don't know why it's in the search/ directory, and base.py is misleading. out of the directories we have, i would think a new file (`test_classes.py`, et al) in `micromasters/tests` would make more sense, but maybe we want a new directory